lib/assets/SnapshotHelper.swift

```diff
@@ -37,29 +46,51 @@ func snapshot(name: String, waitForLoadingIndicator: Bool = true)
             print("Couldn't detect/set language...")
         }
     }
-    
```

Comment:
If you don't mind, could you just clean up these spacing changes? It just makes sure that we don't have to deal with spacing diffs back and forth down the line. :smile: